messages/agent.create.md

```diff
@@ -1,25 +1,25 @@
 # summary
 
-Create an Agent from an agent spec.
+Create an agent in your org from a local agent spec file.
 
 # description
 
-Create an Agent from an agent spec. Agent metadata is created in the target org and retrieved to the local project.
+To generate an agent spec file, run the "agent generate spec" CLI command, which outputs a JSON file with the list of jobs and descriptions that the new agent can perform. Then specify this generated spec file to the --job-spec flag of this command, along with the name of the new agent.
 
-# flags.job-spec.summary
+When this command finishes, your org contains the new agent, which you can then edit in the Agent Builder UI. This command also retrieves all the metadata files associated with the new agent to your local DX project.
 
-The path to an agent spec file.
+To open the new agent in your org's Agent Builder UI, run this command: "sf org open agent --name <api-name-of-your-agent>". If your agent's name includes spaces, replace them with underscores to get the API name. For example, if you specified --name "My Agent", the API name is "My_Agent".
 
-# flags.job-spec.description
+# flags.job-spec.summary
 
-The agent spec file defines job titles and descriptions for the agent and can be created using the `sf agent create spec` command.
+Path to an agent spec file.
 
 # flags.name.summary
 
-The name of the agent.
+Name of the new agent.
 
 # examples
 
-- Create an Agent:
+- Create an agent called "Customer Support Agent" in an org with alias "my-org" using the specified agent spec file:
 
-  <%= config.bin %> <%= command.id %> --spec ./agent-spec.json
+  <%= config.bin %> <%= command.id %> --name "Customer Support Agent" --job-spec ./config/agentSpec.json --target-org my-org
```

Comment:
this command will fail, the name _is_ the API name, and can't contain spaces

Reply:
Oh, huh.  Okay.  Seems weird to pass an API name.  So there's no way to get a more user-friendly name?

Reply:
What I mean is that when they create the agent, in the UI the non-API name can never have spaces?

Reply:
I updated the help.